conda_build/api.py

```diff
@@ -395,8 +395,8 @@ def update_index(dir_paths, config=None, force=False, check_md5=False, remove=Fa
                      subdirs=ensure_list(subdir), convert_if_not_present=convert_if_not_present)
 
 
-def debug(recipe_or_package_path_or_metadata_tuples, path=None, test=False, output_id=None, config=None,
-          verbose=True, **kwargs):
+def debug(recipe_or_package_path_or_metadata_tuples, path=None, test=False,
+          link_source_method='auto', output_id=None, config=None, verbose=True, **kwargs):
```

Comment:
When adding new arguments like this, it's better to put them at the end (here, between verbose=True and **kwargs).  Keyword arguments can still be passed positionally, and we should avoid accidentally breaking people who choose to only do positional args (crazy though they may be).